<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettings</class>
 <widget class="QDialog" name="ConnectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verbindungseinstellungen</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboBox_Protokoll">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Protokoll</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lokale MPI</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU MPI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CPU_MPI">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0 - 127</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP-Adresse</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Speed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Speed 9k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Speed 19k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Speed 187k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Speed 500k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Speed 1500k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Speed 45k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Speed 93k</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Geschwindigkeit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IP">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>xxx.xxx.xxx.xxx</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_local_MPI">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0 - 127</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rack</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Slot</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Rack">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0 - 127</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Slot">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0 - 127</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>461</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Speicherbereich</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBoxArea_0"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditBit_0"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditAddress_0"/>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="comboBoxFormat_0"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Repr√§sentation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Bit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditAddress_1"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditAddress_2"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditAddress_3"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditAddress_4"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEditBit_1"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEditBit_2"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEditBit_3"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEditBit_4"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBoxArea_1"/>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBoxArea_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="comboBoxArea_3"/>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="comboBoxArea_4"/>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="comboBoxFormat_1"/>
    </item>
    <item row="3" column="3">
     <widget class="QComboBox" name="comboBoxFormat_2"/>
    </item>
    <item row="4" column="3">
     <widget class="QComboBox" name="comboBoxFormat_3"/>
    </item>
    <item row="5" column="3">
     <widget class="QComboBox" name="comboBoxFormat_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Operanden</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
